### PR TITLE
feat: respect assist

### PR DESCRIPTION
**Pull request by mikejgray**
Home Assistant now allows you to explicitly expose entities to specific voice assistants.

This feature will respect that setting by default.

Addresses #37

### CODE REVIEW

**Review by NeonDaniel — COMMENTED** [review]
Is the default HomeAssistant behavior documented somewhere (maybe worth a note or link in the README)? I will do some testing to make sure the default behavior works as expected

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ The config also takes some optional properties:
 
 `brightness_increment` - the amount to increment/decrement the brightness of a light when the brightness up/down commands are sent. The default value is 10 and represents a percentage, e.g. 10%.
 `search_confidence_threshold` - the confidence threshold for the search skill to use when searching for devices. The default value is 0.5, or 50%. Must be a value between 0 and 1.
+`assist_only` - whether to pull down only entities exposed to Assist. Default True.
```
Comment (mikejgray) [inline README.md:63]: @NeonDaniel  it's documented here. We could probably do a better job though - any thoughts?

Comment (NeonDaniel) [reply README.md:63]: I was more wondering if HomeAssistant will expose entities to assist by default; i.e. do I need to go pick which things I want exposed to this plugin, or pick the ones not to expose

Comment (mikejgray) [reply README.md:63]: Ah, yes, it generally does expose things to Assist by default. I've pared mine down quite a bit so this PR is a bit selfish. Hoping to have fewer false IDs from fuzzy search by having a more targeted list 🙂

**Comment by Ramblurr** [comment]
I would really like to get this feature landed, my HA has so many entities it bogs down my OVOS instance.

Is there any work on this PR that needs doing? I'll give it a test locally

**Comment by mikejgray** [comment]
> I would really like to get this feature landed, my HA has so many entities it bogs down my OVOS instance.
> 
> 
> 
> Is there any work on this PR that needs doing? I'll give it a test locally 

We're just waiting on some testing. I've run it at home and it seems to work but looking for independent confirmation. Please let me know how this branch works for you.

**Review by NeonDaniel — APPROVED** [review]
Tested and seems to be working well for me ignoring devices that I disabled assist for.

On a side-note, it does match devices that are unavailable but I'm not sure if there's a way to handle that and it really is an edge case (I added my lights to a generic Zigbee hub and disabled the Hue integration but have not yet removed it).

**Pull request merged**